lib/mjsonwp/mjsonwp.js

```diff
@@ -189,6 +189,14 @@ function getResponseForJsonwpError (err) {
   return [httpStatus, httpResBody];
 }
 
+function getResponseForW3CError (err) {
+  let httpStatus = err.w3cStatus || 400;
```

Comment:
`400` seems to be known in too many places. Only the error module should know it, really. It would be nicer to either have a guarantee that `err.w3cStatus` is set (could make it a getter on the base error object to ensure that something is returned if nothing is there when it is accessed), or as a constant exported from the error module.